packages/protocol/scripts/check-backward.ts

```diff
@@ -43,6 +45,11 @@ const argv = yargs
     default: false,
     type: 'boolean',
   })
+  .option('new_branch', {
+    alias: 'nb',
```

Comment:
Total nit, but usually aliases are single letter. Not even sure if the library will correctly handle this? Would just go with no alias for this flag.